package.json

```diff
@@ -18,7 +19,9 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged"
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
```

Comment:
That's awesome 👍

Reply:
Its super helpful since the release of the project depends on proper commit messages.